core/src/components/button/button.tsx

```diff
@@ -128,7 +128,7 @@ export class Button implements ComponentInterface, AnchorInterface, ButtonInterf
    * is inside of an item, where the size is `"small"` by default. Set the size to
    * `"default"` inside of an item to make it a standard size button.
    */
-  @Prop({ reflect: true }) size?: 'xsmall' | 'small' | 'default' | 'large' | 'xlarge';
+  @Prop({ reflect: true }) size?: 'small' | 'default' | 'large';
```

Comment:
```suggestion
  @Prop({ reflect: true }) size?: 'small' | 'medium' | 'large';
```

Should we change `default` to `medium` to be consistent with other components? This would be breaking and we would have to update it in other places. Otherwise we could just add `medium` here for now.

Reply:
Added the medium:
** @Prop({ reflect: true }) size?: 'small' | 'default' | 'medium' | 'large';**